package.json

```diff
@@ -11,20 +11,21 @@
     "build:widgets": "cross-env NODE_ENV=${NODE_ENV:-production} npm run build widgets",
     "build:sri": "npm run build sri",
     "clean": "npm run clean:dist && npm run clean:transpile",
-    "clean:dist": "rimraf \"packages/**/dist/\"",
+    "clean:dist": "rimraf \"packages/**/dist/\" && rimraf \"packages/**/dist/\"",
     "clean:transpile": "rimraf \"packages/**/es/\" && rimraf \"packages/**/cjs/\"",
     "test": "npm run static-analysis && npm run jest",
     "test:clean": "npm run clean && npm run test",
-    "test:automation": "wdio wdio.conf.js",
-    "test:automation:oneOnOne": "wdio wdio.conf.js --suite oneOnOne",
-    "test:automation:space": "wdio wdio.conf.js --suite space",
-    "test:automation:recents": "wdio wdio.conf.js --suite recents",
-    "test:tap": "cross-env TAP=true wdio wdio.conf.js --suite tap",
-    "test:integration": "cross-env INTEGRATION=true wdio wdio.conf.js --suite integration",
+    "test:automation": "wdio ./wdio/wdio.conf.js",
+    "test:automation:oneOnOne": "wdio ./wdio/wdio.conf.js --suite oneOnOne",
+    "test:automation:space": "wdio ./wdio/wdio.conf.js --suite space",
+    "test:automation:recents": "wdio ./wdio/wdio.conf.js --suite recents",
+    "test:tap": "wdio ./wdio/wdio.tap.conf.js --suite tap",
+    "test:integration": "cross-env INTEGRATION=true wdio ./wdio/wdio.conf.js --suite integration",
+    "test:integration:sauce": "cross-env INTEGRATION=true wdio ./wdio/wdio.sauce.conf.js --suite integration",
```

Comment:
needs `SAUCE=true` right?

Reply:
I tried to remove all the flags that have logic into the `wdio.conf.js` to have specific files based on the the type of test or environment you're running in. That way we don't have to manage extra flags. `INTEGRATION` was one I couldn't get rid of because it's used in logic in our actual test suites. May want to move that out into a `pre` hook.